pkg/console/util.go

```diff
@@ -647,19 +647,7 @@ func validateDiskSize(devPath string) error {
 		return err
 	}
 	if diskSizeBytes>>30 < config.HardMinDiskSizeGiB {
-		return fmt.Errorf("Disk size too small. Minimum %dGB is required", config.HardMinDiskSizeGiB)
-	}
-
-	return nil
-}
-
-func validateDiskSizeSoft(devPath string) error {
-	diskSizeBytes, err := util.GetDiskSizeBytes(devPath)
-	if err != nil {
-		return err
-	}
-	if diskSizeBytes>>30 < config.SoftMinDiskSizeGiB {
-		return fmt.Errorf("Disk size is smaller than the recommended size: %dGB", config.SoftMinDiskSizeGiB)
+		return fmt.Errorf("Disk size is too small. Minimum %dGi is required", config.HardMinDiskSizeGiB)
```

Comment:
For the additional data disk, the hard limit can probably be set to `50Gi` since users can add additional disks to the LH later.